OpenRA.Mods.Common/Traits/Attack/AttackFollow.cs

```diff
@@ -84,7 +84,7 @@ public override Activity Tick(Actor self)
 				if (IsCanceled || !target.IsValidFor(self))
 					return NextActivity;
 
-				if (self.IsDisabled())
+				if (attack.IsTraitDisabled)
```

Comment:
AttackBase maps `IsDisabled()` &rarr; `IsTraitPaused`, so shouldn't that be the same here?